config/initializers/figaro.rb

```diff
@@ -1,9 +1,11 @@
 # config/initializers/figaro.rb
 
-Figaro.require_keys("S3_KEY",
-                    "S3_SECRET",
-                    "S3_BUCKET",
-                    "S3_REGION",
-                    "EMAIL_USERNAME",
-                    "EMAIL_PASSWORD"
-                    )
+Figaro.require_keys(
+  "EMAIL_USERNAME",
+  "EMAIL_PASSWORD",
```

Comment:
As far as I know there isn't a way to set up Figaro like this, which I think makes sense since the point of something like Figaro is to not have different environment variables on different environments